test/CustomMatchers.test.js

```diff
@@ -43,5 +43,23 @@ define([], function () {
         };
     };
 
+    CustomMatchers.toEqualSector = function (util, customEqualityTesters) {
```

Comment:
Should this be a fuzzy comparison, or are the sectors we're checking in the unit test exact? Said another way, would this be better as `toBeSector`?

I have upcoming changes to CustomMatchers that adds `toBeVec3`, `toBeCloseToVec3` `toBePosition`, and `toBeCloseToPosition`.